packages/ui-a11y-utils/package.json

```diff
@@ -29,7 +29,7 @@
     "@instructure/ui-dom-utils": "9.2.0",
     "@instructure/ui-react-utils": "9.2.0",
     "@instructure/uid": "9.2.0",
-    "keycode": "^2.2.1"
```

Comment:
Its better to define supported versions for user-consimed packages in the broadest range possible to reduce the chance of including it twice in their builds. Luckily we just have a few such dependencies